projects/plugins/jetpack/jetpack.php

```diff
@@ -16,6 +16,10 @@
 
 use Automattic\Jetpack\Image_CDN\Image_CDN_Core;
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit( 0 );
+}
+
 /*
```

Comment:
If we're going to have a GPL header comment (versus just the `License: GPL2+` header line), I wonder if it should go above the `use` and the guard rather than after.

Reply:
15272b7be46